src/app/space/create/deployments/apps/deployment-card.component.ts

```diff
@@ -59,32 +60,35 @@ export class DeploymentCardComponent implements OnDestroy, OnInit {
     this.iconClass = DeploymentStatusIconComponent.CLASSES.ICON_OK;
     this.toolTip = 'Everything is ok';
 
-    this.cpuStat = this.deploymentsService.getDeploymentCpuStat(this.spaceId, this.applicationId, this.environment.name);
-    this.subscriptions.push(this.cpuStat.subscribe((stat) => {
-      this.changeStatus(stat);
+    this.subscriptions.push(this.spaceId.subscribe((spaceId: string) => {
+      this.spaceIdReference = spaceId;
+      this.cpuStat = this.deploymentsService.getDeploymentCpuStat(spaceId, this.applicationId, this.environment.name);
+      this.subscriptions.push(this.cpuStat.subscribe((stat) => {
+        this.changeStatus(stat);
+      }));
+
+      this.subscriptions.push(
+        this.deploymentsService
+          .isApplicationDeployedInEnvironment(spaceId, this.applicationId, this.environment.name)
+          .subscribe((active: boolean) => {
+            this.active = active;
+
+            if (active) {
+              this.version =
+                this.deploymentsService.getVersion(spaceId, this.applicationId, this.environment.name);
+
+              this.logsUrl =
+                this.deploymentsService.getLogsUrl(spaceId, this.applicationId, this.environment.name);
+
+              this.consoleUrl =
+                this.deploymentsService.getConsoleUrl(spaceId, this.applicationId, this.environment.name);
+
+              this.appUrl =
+                this.deploymentsService.getAppUrl(spaceId, this.applicationId, this.environment.name);
```

Comment:
cpuStat, version, and the URL Observables would ideally also be backed by Subjects here. This can be done in a follow-up commit on top of this patch series